zaza/openstack/charm_tests/keystone/tests.py

```diff
@@ -380,7 +380,12 @@ def test_security_checklist(self):
 
 
 class LdapTests(BaseKeystoneTest):
-    """Keystone ldap tests tests."""
+    """Keystone ldap tests."""
+
+    special_type_keys = ('ldap-user-enabled-mask',
+                         'ldap-user-enabled-invert',
+                         'ldap-group-members-are-ids',
+                         'ldap-use-pool')
```

Comment:
What makes a key 'special'?  I'm assuming it's something to do with a config value being 'None' or '' which means it doesn't override something in the ldap config key (i.e. there are two ways of setting things).  But it's not clear what 'special' means here.